Scripts/Python/xGUILinkHandler.py

```diff
@@ -53,8 +53,12 @@
 
 from xCensor import xCensor
 
-_HOST_REGEX = re.compile(R"https?:\/\/(?:www\.)?([^\/]+).*", re.IGNORECASE)
-_URL_REGEX = re.compile(R"https?:\/\/[\S]+", re.IGNORECASE)
+# These regexes match URLs in the format of:
+# https://(anything not a space
+# http://(anything not a space)
+# www.(anything not a space or a dot).(anything not a space)
+_HOST_REGEX = re.compile(R"(?:https?:\/\/)?(?:www\.)?([^\/]+).*", re.IGNORECASE)
```

Comment:
You may also want to strip of fragments and queries, if they appear before the first `/`
```suggestion
_HOST_REGEX = re.compile(R"(?:https?:\/\/)?(?:www\.)?([^\/?#]+).*", re.IGNORECASE)
```

Also, I don't see a problem with leaving `www.` on the prompted hostname, unless we're worried it will cause the message to get too long.